content/blog/wingingit/winging-it-14.md

```diff
@@ -0,0 +1,54 @@
+---
+title: CSS IS AWESOME Game
+episode: 14
+tags:
+  - CSS
+author:
+  - miriam
+  - stacy
+date: 2024-12-17
+length: 90 mins
+image:
+  src: winging-it/winging-it-14.jpg
+media:
+  - youtube: fYfs6sKGciQ
+summary: |
+  On December 17, 1996,
+  the W3C published the first standard for CSS,
+  and we celebrated the anniversary on Winging It!
+  [Michelle Barker](https://michellebarker.co.uk/),
+  [Stephanie Eckles](https://thinkdobecreate.com/),
+  [Kevin Powell](https://www.kevinpowell.co/),
+  and [Henri Helvetica](https://bsky.app/profile/henrihelvetica.bsky.social)
+  joined us for our "CSS Is Awesome" game.
+  If you love CSS as much as we do,
+  we hope you will play along
+  as we battle it out
+  to see who is AWESOME.
+---
+
+{% import "embed.macros.njk" as embed %}
+{% import "utility.macros.njk" as utility %}
+
+{{ embed.figure(
+  data=media
+) }}
+
+{{ utility.main_action(
+  'Subscribe to Channel »',
+  subscribe_url
+) }}
+
+## Games we play:
+
+- 7-Point Polygon
+- Password Reset
+- Jeoparty
```

Comment:
Is this an intentional misspelling?

Reply:
It is, yes.

Reply:
I take it you didn't watch the stream 🤣

Reply:
I actually did! But I must have missed or forgotten that 😄